include/picongpu/simulation/stage/AtomicPhysics.x.cpp

```diff
@@ -107,7 +107,7 @@ namespace picongpu::simulation::stage
                 = particles::atomicPhysics::localHelperFields::FoundUnboundIonField<picongpu::MappingDesc>;
 
             // species Lists
-            //{
+            //@{
```

Comment:
only a note: This and the line above must be a doxygen comment e.g. `///` or `//!` or `/**`

Reply:
fixed